src/library-authoring/containers/ContainerInfo.tsx

```diff
@@ -2,13 +2,13 @@ import { FormattedMessage, useIntl } from '@edx/frontend-platform/i18n';
 import {
   Button,
   Stack,
-  Tab,
-  Tabs,
   Dropdown,
   Icon,
   IconButton,
   useToggle,
 } from '@openedx/paragon';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Tab, Nav } from 'react-bootstrap';
```

Comment:
Same comment as the [other PR](https://github.com/openedx/frontend-app-authoring/pull/2475#discussion_r2434126173):

> We should not be directly importing from `react-bootstrap`. That's what the eslint warning `import/no-extraneous-dependencies` is trying to tell you.
>
> Future versions of Paragon may be refactored to not use bootstrap at all, so we should only use Paragon APIs.